src/page-templates/content-page/content-page-template.tsx

```diff
@@ -395,7 +396,17 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
                             />
                         )}
                         {codeType && (
-                            <SecondaryTag codeLevel={codeType as CodeLevel} />
+                            <SecondaryTag
```

Comment:
i remember coming across this change when I was working on the event detail page because I think there was a TS error on the `icon` prop, did you specifically make a change in this file? Just asking because I think some weird versioning issue happened here...if you did then ignore this but if you didn't then you can safely just revert this whole file

Reply:
yeah I made this change--before, the SecondaryTag was coupled to the code level type cards, but since now the events cards also need to display a secondary tag, I generalized the component so it could be used by both which necessitated an update here

Reply:
ok cool, that was probably leftover from when i originally pulled down your branch in the beginning. just wanted to make sure